components-core/src/main/java/com/adyen/checkout/components/core/internal/ui/ViewableDelegate.kt

```diff
@@ -6,10 +6,13 @@
  * Created by josephj on 24/8/2022.
  */
 
-package com.adyen.checkout.components.base
+package com.adyen.checkout.components.core.internal.ui
```

Comment:
I think this should be inside ui-core module WDYT?

Reply:
I think we should leave it here. `ui-core` is more "real" UI related, so custom views and styling. There are some not so UI related classes in there, but let's try and keep that to a minimum